### ISSUE TITLE
sysusers fails 

### ISSUE THREAD

**Issue by udeved** [artix-linux/opensysusers#2]
I was trying to install jenking on the server.

```
(30/31) installing jenkins                                            [#######################################] 100%
groupadd: invalid group ID '-'
useradd: invalid user ID '-'
groupadd: invalid group ID '-'
install: invalid user 'jenkins'
error: command failed to execute correctly
(31/31) installing jenkins-openrc                                     [#######################################] 100%
:: Running post-transaction hooks...
(1/4) Updating fontconfig cache...
(2/4) Displaying openrc service help ...
        ==> Add a service to runlevel:
        rc-update add <service> <runlevel>
(3/4) Updating system users and groups
groupadd: invalid group ID '-'
useradd: invalid user ID '-'
groupadd: invalid group ID '-'

```

The jenkins sysusers dile looks like this.

```
u jenkins - "Jenkins CI" /var/lib/jenkins
g jenkins -
```

**Comment by udeved** [comment]
I think we need a more dynamic approach to useradd and groupadd parameters.

**Comment by cromerc** [comment]
I see the issue with the jenkins sysuser.d file, I will fix this tomorrow morning after work.

**Comment by udeved** [comment]
sysusers seems to have some more abilty.

Take this jenkins scriptlet. They call it despite the alpm hook.

```bash
post_install() {
  # We need to know uid and gid to create /var/lib/jenkins with proper
  # permissions. So keep systemd-sysusers invocation here despite having
  # pacman hooks.
  systemd-sysusers jenkins.conf
  [[ -d var/lib/jenkins ]] || install -dm 750 -o jenkins -g jenkins var/lib/jenkins
}
```


The comment would suggest, systemd-sysusers matches gid and uid against their names.
But why it calls it with the actualy service conf is a mystery to me.

The service conf looks like this, installed in /etc/conf.d/jenkins
```

JAVA=/usr/bin/java
JAVA_ARGS=-Xmx512m
JAVA_OPTS=
JENKINS_USER=jenkins
JENKINS_HOME=/var/lib/jenkins
JENKINS_WAR=/usr/share/java/jenkins/jenkins.war
JENKINS_WEBROOT=--webroot=/var/cache/jenkins
JENKINS_PORT=--httpPort=8090
JENKINS_AJPPORT=--ajp13Port=-1
JENKINS_OPTS=
JENKINS_COMMAND_LINE="$JAVA $JAVA_ARGS $JAVA_OPTS -jar $JENKINS_WAR $JENKINS_WEBROOT $JENKINS_PORT $JENKINS_AJPPORT $JENKINS_OPTS"

```

**Comment by cromerc** [comment]
Closed via 4034a90d2aa08ab79b571509fb6d094ddff0b882
